### ISSUE TITLE
[Bug]: Will lag firing sound of gun mod

### ISSUE THREAD

**Issue by Sorrow-Scarlet** [someaddons/performant_issues#278]
### Describe the bug you're experiencing

i have mod timeless and classic mod, performant. This mod will actually "eat" firing sound.

### Reproducability

1. install timeless and classic/ Mr.Crayfish's gun or any additional gun mod of Crayfish's, install performant
2. keep everything default.
3. find any automatic weapon
4. fire and listen.

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.16.5

### Modloader version

Forge

### Logs

game is not crashed, sry bro.

**Comment by someaddons** [comment]
performant prevents the same soundsource from playing more than 10 times concurrently, probably that mod does something odd with its sounds

**Comment by Sorrow-Scarlet** [comment]
Yes, i wonder if you can add a switch to control whether to use this improvement, that would help.
